COBISS.js

```diff
@@ -0,0 +1,1098 @@
+	var catalogItemTypeHash = new Map([
+		['undergraduate thesis', 'thesis'],
+		['proceedings', 'conferencePaper'],
+		['novel', 'book'], // https://plus.cobiss.net/cobiss/si/en/bib/35544323
+		['science fiction (prose)', 'book'], //https://plus.cobiss.net/cobiss/si/en/bib/46310659
+		['book', 'book'],
+		['handbook', 'book'],
+		['proceedings of conference contributions', 'conferencePaper'], //https://plus.cobiss.net/cobiss/si/en/bib/91188227
+		['professional monograph', 'report'], //https://plus.cobiss.net/cobiss/si/en/bib/87583747
+		['scientific monograph', 'book'],
+		['textbook', 'book'],
+		['e-book', 'book'],
+		['picture book', 'book'],
+		['treatise, study', 'report'],
+		['catalogue', 'book'],
+		['master\u0027s thesis', 'thesis'],
+		['picture book', 'book'],
+		['short stories', 'book'],
+		['research report', 'report'],
+		['poetry', 'book'],
+		['dissertation', 'thesis'],
+		['picture book', 'book'],
+		['offprint', 'magazineArticle'],
+		['guide-book', 'book'],
+		['expertise', 'hearing'], // this is court testimony, not sure what the correct item type should be https://plus.cobiss.net/cobiss/si/en/bib/94791683
+		['profess. monogr', 'report'],
+		['project documentation', 'report'],
+		['antiquarian material', 'book'], // most of these are books, e.g. https://plus.cobiss.net/cobiss/si/en/bib/7543093
+		['other lit.forms', 'book'],
+		['drama', 'book'],
+		['strip cartoon', 'book'],
+		['documentary lit', 'book'],
+		['encyclopedia', 'book'],
+		['exercise book', 'book'],
+		['educational material', 'book'],
+		['review', 'report'],
+		['statistics', 'report'],
+		['legislation', 'statute'],
+		['essay', 'book'],
+		['final paper', 'thesis'],
+		['standard', 'book'],
+		['specialist thesis', 'book'],
+		['aphorisms, proverbs', 'book'],
+		['humour, satire, parody', 'book'],
+		['examin. paper', 'report'],
+		['annual', 'report'],
+		['yearly', 'report'],
+		['documentary lit', 'book'],
+		['folk literature', 'book'],
+		['patent', 'patent'],
+		['regulations', 'report'],
+		['conf. materials', 'conferencePaper'],
+		['radio play', 'book'],
+		['letters', 'book'],
+		['literature survey/review', 'report'],
+		['statute', 'statute'],
+		['matura paper', 'thesis'],
+		['seminar paper', 'thesis'],
+		['habilitation', 'thesis'],
+		['dramaturgical paper', 'thesis'],
+		['', ''],
+		['', ''],
+		['', ''],
+		['', ''],
+		['', ''],
+
+		// TODO: finish for remaining types in search all. Not all will have RIS.
+
+	]);
+	return (catalogItemTypeHash.get(englishCatalogItemType));
+}
+
+async function doWeb(doc, url) {
+	// var fullRecordURL = "https://plus.cobiss.net/cobiss/si/en/bib/COBIB/93266179/full"
+	var fullRecordURL = "https://plus.cobiss.net/cobiss/si/en/bib/93266179#full"
+	if (detectWeb(doc, url) == 'multiple') {
+		let items = await Zotero.selectItems(getSearchResults(doc, false));
+		if (!items) return;
+		for (let url of Object.keys(items)) {
+			await scrape(await requestDocument(url), await requestDocument(fullRecordURL));
```

Comment:
@AbeJellinek this is what's not working currently

Reply:
The URL is using content negotiation, so when you request it from the browser it returns HTML, but otherwise it returns JSON. That's actually better for us, but `requestDocument()` won't work. `requestJSON()` should get you the JSON.

Reply:
that worked, thanks!